sdk/keyvault/azure-keyvault-keys/src/samples/java/com/azure/security/keyvault/keys/KeyClientJavaDocCodeSnippets.java

```diff
@@ -352,6 +352,18 @@ public void listKeySnippets() {
                 keyWithMaterial.keyMaterial().kty());
         }
         // END: com.azure.keyvault.keys.keyclient.listKeys#Context
+
+        // BEGIN: com.azure.keyvault.keys.keyclient.listKeys.iterableByPage
+        keyClient.listKeys().iterableByPage().forEach(resp -> {
+            System.out.printf("Response headers are %s. Url %s  and status code %d %n", resp.headers(),
+                resp.request().url(), resp.statusCode());
+            resp.items().forEach(value -> {
+                Key keyWithMaterial = keyClient.getKey(value);
```

Comment:
Do we want to make a call to get each key again? Why not just print the name?

Reply:
All other samples are making this extra call for many APIs. Not doing here would be odd.